### ISSUE TITLE
Is it just an exercise or a real thing?

### ISSUE THREAD

**Issue by qur2** [jas-chen/rx-redux#6]
Hi,

I stumbled on your repo(s) through the redux awesome list and I find your experiments quite interesting. I'm wondering (specifically about this one, after checking Cycle.js) if you plan any serious development or not really.

Thanks!

**Comment by jas-chen** [comment]
Hi,

`rx-redux` is just an exercise at the beginning, but during the re-implementation I found out that middleware is not suit for doing async in the RxJS stream approach, and that is why there is RxMiddleware in this project, to make it all the way streaming.

After that I realize the re-implementation of all the redux APIs is meaningless, so I created redux-core,  which is redux without middleware functions.

If you want to use redux and RxJS in your serious production app, you have two options:
1. Use redux (because it is well tested) but don't do async operations in middleware. Examples can be found in [thisless-react](https://github.com/jas-chen/thisless-react).
2. Use reducer only, put it into RxJS scan(). The code will be like this:

``` javascript
const intent$ = getIntentStreamSomehow();
const action$ = intent$.flatMap(transformToActionStream); // RxMiddleware in mind
const state$ = action$.scan(reducer, initState);

state$.subscribe(render);
```
